### PR TITLE
Add share button

### PR DESCRIPTION
**Pull request by mojowen**
- [x] Implement HTML2Canvass with the D3 Map
- [x] Create share modal or something after the share button is clicked

### CODE REVIEW

**Comment by mojowen** [comment]
@noahmanger  this is ready to go - I'll do the uploading in a follow up.

image

File: src/content/index.js [hunk 1]
```diff
@@ -4,6 +4,28 @@ import { legendWidth, legendHeight, prefix } from "../constants";
 
 import { buildSheetsURL } from "../utils";
 
+let vis;
+let shareContainer;
+
+export const toggleLoading = (isLoading, elem = vis) => {
```
Comment (mojowen) [inline src/content/index.js:11]: this will enable the loading overlay

File: styles/loader.scss [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+       &:nth-child(#{$i}) {
+         margin-left: ($i - 1) * ($width + $border);;
+         animation: barUp#{$i} 2s infinite;
+        }
+    }
+
+  }
+}
+
+@keyframes barUp1 {
```
Comment (noahmanger) [inline styles/loader.scss:38]: This is so nice.

**Review by noahmanger — APPROVED** [review]
Nice! This works great. I pushed a couple small styling fixes.

One thing I noticed but couldn't figure out is that when you generate the screenshot, the height of the SVG changes. And on mobile, it really elongates it.

**Pull request merged**